teamwork.css

```diff
@@ -1,10 +1,13 @@
-.container {
-  margin-left: 150px;
-  margin-right: 150px;
+.Content-2-main-container {
+ display: flex;
+ flex-direction: column;
+ align-items: center ;
+ justify-content: space-around;
 }
 
 body {
```

Comment:
styles with body should be in one class . you can grab it from all files and move to style.css which is general file